### ISSUE TITLE
Could you clarify the license / your relationship to Outrun Labs?

### ISSUE THREAD

**Issue by donaldguy** [ospfranco/tempomat#2]
Hi Oscar,

I am confused about the license terms of this software (which I bought off the Mac app store earlier today, and may be interested in making some functional improvements to)

Having looked at your CV from your website, it does not appear that you are an employee or legal agent of Outrun Labs, and I want to get clarification about this—and warn you about the potential mistake you have made / are making if indeed you are not.

--- 

It is my guess that you in good faith included the license under the understanding that it offered the sort of terms you wished to grant*, however—unlike the various standard open source licenses, and indeed unlike some shared source licenses ([the Fair Source License](https://fair.io/?a#license) for example)—**the Outrun Labs EULA [as included in this repo](https://github.com/ospfranco/Tempomat/blob/05748fd3c5548a1f897df821345ba5f2e73ffa97/LICENSE.md) is NOT written as a general contract between parties**, it is explicitly and **_only_** an agreement between the acceptor (me) and Outrun Labs LLC.

To quote (emphasis added):

>Subject to terms and the scope of the applicable license model as set out in clause 2, **Outrun Labs grants You** a limited, revocable, non-exclusive, non-transferable license to download, install, and use a machine-readable, object code version of the software programs purchased by Licensee ("Software") [...] however, that Licensee's right to download, install, and use the Software [...] is limited to those rights expressly set out in this EULA.

If you are not a legal agent of Outrun Labs, you have no legal capability to make such a grant on their behalf (nor if you have not assigned them the copyright herein**, do they have the right to make such a grant on this application). 

**Meanwhile you (understood as Oscar Franco, an individual, the apparent holder of copyright), have made no clear grants whatsoever**. (except insofar as copies bought on the mac app store have a license granted for use by the terms of [the Mac App Store EULA](https://www.apple.com/legal/macapps/stdeula/))

 As such, **if you are not a legal agent of Outrun Labs, while the license remains as is, you have effectively made it not clearly legal for anyone to use this software at all** 

(except possibly when they buy it on the Mac App Store. I'd worry, however, that Apple will also be concerned at some point about the incompatible and apparently exclusive terms being offered here, which forbid all commercial use under 1.4d; and then (without clear language specifying how a commercial license supersedes that provision, _possibly_) allow for commercial use only insofar as it is bought _on a recurring subscription basis, and bought **from Outrun Labs**_  )

--- 

So, in short, if you are not Outrun Labs, you should think about changing the license ASAP (and if you are, you should clarify how)

---

*free for non-commercial use, license required for commercial use, source code available and usable for personal modification, but not re-licensable– a fact that doesn't strictly need stating insofar as your exclusive rights are reserved under most copyright laws unless and until you issue other license

**I am going to chose to read the inclusion of the license file as not being a grant of that copyright to Outrun Labs, though section 4.1's inclusion could be understood to

**Comment by donaldguy** [comment]
I would also ask you to consider what you are hoping to achieve by making source of the application available (which I appreciate) 

The Outrun Labs license fairly clearly grants provision to make modifications for personal use, but it seems like it probably would violate the license for anyone to push any code to a fork, let alone open a PR to you the upstream 

I personally would like to improve the application, but really only if the time I spend making changes have the potential to benefit people other than just me.

**Comment by donaldguy** [comment]
I'd suggest looking at https://sfosc.org/docs/ while you think about whats an appropriate license (unless, again, you are actually a legal agent of Outrun Labs – in which case, you can mostly ignore me :-) )

**Comment by ospfranco** [comment]
I'm not related to Outrun Labs, I just took the license from them, the legal text inside of it might not be 100% correct/clear, however there are other open source projects out there using it (ex. [onivim](https://github.com/onivim/oni2)), so I thought there would be no problems.

In any case, I can take a look later but as stated in the README you can contribute code and PRs and use it personally (since I cannot stop you anyways) but trying to commercialize a copy of the app under a different name is not allowed.

**Comment by donaldguy** [comment]
Onivim is a/the product of Outrun Labs
![image](https://user-images.githubusercontent.com/43136/93606496-289cd980-f996-11ea-865f-aa49abce3693.png)

**Comment by ospfranco** [comment]
Aha, I did not realize it.

Ok I will change the license later then, thanks for noticing

**Comment by ospfranco** [comment]
I briefly took down the repo until I had the time to read a bit upon licenses, I have gone with the fair source license but did a slight modification to prevent any distribution whatsoever, I think this is enough for my purposes (make money of the software I created) while still allowing other people to contribute.
